README.md

```diff
@@ -3,17 +3,10 @@ Kill Bill is an open-source billing and payments platform
 
 This Python package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
 
-- API version: 0.22.22
+- API version: 0.24.0
 - Package version: 0.0.1-SNAPSHOT
 - Build package: org.killbill.billing.codegen.languages.KillbillPythonGenerator
 
-## Versions
```

Comment:
I would keep this section and indicate the minimum Python version for 0.24.x. Is it Python 3.x?